docs/changelog.md

```diff
@@ -1,5 +1,8 @@
 ## Next version
 
+## Next version
+* (Major) Support project-based feature flag loading.
```

Comment:
Is this breaking somehow? I would argue it is a minor (feature release).

Reply:
Oops, yus that should be a minor bump.

Reply:
Oops, that should be a minor bump.